test/org/sagebionetworks/bridge/models/schedules/TaskSchedulerTest.java

```diff
@@ -76,10 +82,9 @@ public void taskIsComplete() {
         schedule.addActivity(TestConstants.TEST_ACTIVITY);
         schedule.setExpires("P3Y");
         
-        tasks = SchedulerFactory.getScheduler("schedulePlanGuid", schedule).getTasks(events, NOW.plusWeeks(1));
+        tasks = schedule.getScheduler().getTasks(getContext(NOW.plusWeeks(1)));
         Task task = tasks.get(0);
 
-        assertEquals("schedulePlanGuid", task.getSchedulePlanGuid());
         assertNotNull(task.getGuid());
         assertEquals("Label", task.getActivity().getLabel());
         assertEquals("tapTest", task.getActivity().getRef());
```

Comment:
Test that task contains healthcode and timezone? Test localScheduledOn and localExpiresOn? Run key?